apps/crn-server/test/fixtures/teams.fixtures.ts

```diff
@@ -139,7 +172,19 @@ export const getTeamDataObject = (): TeamDataObject => ({
   tools: [],
 });
 
+export const getTeamReducedDataObject = (): TeamReducedDataObject => ({
```

Comment:
I don't like the name reduced. I think its better to be specific that this is the list response data type. Reduced could be for anything.

Reply:
ref: https://github.com/yldio/asap-hub/pull/3929#discussion_r1408057063